### DISCUSSION TITLE
Welcome to AndroidEasterEggs Discussions!

### DISCUSSION THREAD

**Discussion by hushenghao** [hushenghao/AndroidEasterEggs#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lorypelli** [comment]
What about making a server discord?

**Reply by hushenghao** [reply]
That sounds good, but I probably won't have much time to maintain it

**Reply by simsimient1** [reply]
Discord server is chat?

**Comment by HaCOn86** [comment]
with android 14 beta 4 ,new easter egg released ,i wonder how long it will take you to implement it to the app ? new easter egg look good i cant wait to try it and play it

**Reply by hushenghao** [reply]
I can't give you a definite time, AOSP hasn't released the source code yet, I'll just have to extract the APK from the system and try to decompile it.

**Reply by HaCOn86** [reply]
> I can't give you a definite time, AOSP hasn't released the source code yet, I'll just have to extract the APK from the system and try to decompile it.

take your time 😅

**Reply by hushenghao** [reply]
v1.9.5 has been released, but the decompiled version does not yet support all features.

**Reply by HaCOn86** [reply]
> v1.9.5 has been released, but the decompiled version does not yet support all features.

i installed it yes minigame not work but haptics nice

**Reply by HaCOn86** [reply]
> v1.9.5 has been released, but the decompiled version does not yet support all features.

in this [channel](https://t.me/linuxctchannel) linuxct shared the android 14 easteregg apk and it crashes for me and i  reach out to him and say that app is crashing  [in here](https://t.me/MishaalAndroidNews/609) mishaal s channel and he reach out to me and ask for logs and he say he try to patch so if you want, you can reach out to him to help or get help .🤷🏽‍♂️

**Reply by hushenghao** [reply]
I have also tried to work with older versions of the system, which may have used new features of the system. These attempts are moot until AOSP releases the source code.

**Comment by simsimient1** [comment]
without space one easter egg lol jk
![Screenshot_2023-08-19-17-10-30-817_com.dede.android_eggs-edit.jpg](https://github.com/hushenghao/AndroidEasterEggs/assets/109045595/08cc1285-8434-43f8-8b2c-d496674bf8b0)

to release coming soon of ~~space one~~
SoonTM, in without apps ~~space one~~

**Reply by hushenghao** [reply]
It's just a hint. It's not important. Changing to a dialog will interrupt the user's operation.

**Comment by simsimient1** [comment]
I am fake chat [iChat](https://play.google.com/store/apps/details?id=com.ichatwazzn) worry Redmi 9T **Space One**!

**Reply by simsimient1** [reply]
Device Frame MIUI 15 and screenshot

**Comment by user0-07161** [comment]
When will V1.9.7 be released to F-Droid?

**Reply by hushenghao** [reply]
You can download the beta version to use it first

**Reply by user0-07161** [reply]
ok

**Reply by simsimient1** [reply]
Poyo! Poyo!

**Comment by user0-07161** [comment]


**Reply by hushenghao** [reply]
It's almost impossible for this code to run

**Reply by simsimient1** [reply]
I am so Gradle project

**Comment by The-ProGrammer66** [comment]
When do you usually implement new Crowdin translations into the app?

**Reply by hushenghao** [reply]
Will be uploaded to beta soon, app stores expect delays

**Comment by minimote** [comment]
请问有办法在软件中直接打开彩蛋前的这个页面吗？

![Screenshot_2025-07-27-15-39-03-38_c31b32364ce19ca8fcd150a417ecce58.jpg](https://github.com/user-attachments/assets/d52da0ef-eb26-47d8-a812-db33450e4ad5)


目前用
包名com.android.egg
活动名com.android.egg.landroid.MainActivity
只能进入长按后的彩蛋里
